docs/playbooks/lite-assist-quickburn-external-discovery.md

```diff
@@ -0,0 +1,134 @@
+# MAS Assist Service on DevIT Quickburn
+
```

Comment:
Can we add an opening sentence here that states this is for when CP4D and watson discovery are already setup on an external cluster.

Reply:
updated